akka-cluster/src/main/resources/reference.conf

```diff
@@ -289,5 +289,10 @@ akka {
       adaptive-group = "akka.cluster.routing.AdaptiveLoadBalancingGroup"
     }
   }
+  
+  coordinated-shutdown-phases {
+    cluster-pre-exiting = []
+    cluster-exiting = [cluster-pre-exiting]
+  }
```

Comment:
I'm thinking if we'll be able to provide a default nice shutdown involving many modules, hm...

So let's say we first stop accepting new connections (http), then quit the cluster, then kill http entirely (system), could I express this somehow in reference.conf hm hm...